src/frontend/magnify/src/components/rooms/MyRooms/MyRooms.test.tsx

```diff
@@ -11,4 +12,12 @@ describe('MyRooms', () => {
     // 3) Check the rooms
     expect(screen.getAllByRole('button', { name: 'Join' }).length).toBe(1);
   });
+  it('should display message when rooms list is emty', async () => {
+    render(<MyRooms baseJitsiUrl="" rooms={[]} />);
+    expect(
```

Comment:
No need to put an `expect(...).toBeInTheDocument`. `await screen.findByText()` will crash if it finds nothing